CONTRIBUTING.md

```diff
@@ -7,6 +7,8 @@ To report a bug:
 
 Submit a `1.2.3` version upgrade for the `foo` formula:
 
+- First check if someone else has already submitted the same upgrade, since it can take time to be tested and incorporated. Search the open Pull Requests for `foo` [like this](https://github.com/Homebrew/homebrew/pulls?utf8=✓&q=is%3Apr+is%3Aopen+foo). If nothing relevant shows up, proceed.
```

Comment:
Can be slimmed down a bit: `Check if the same upgrade has been already submitted by [searching the open pull requests for `foo`](https://github.com/Homebrew/homebrew/pulls?utf8=✓&q=is%3Apr+is%3Aopen+foo).`